WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -577,6 +577,15 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
         }
     }
 
+    func gutenbergDidLogUserEvent(_ event: GutenbergUserEvent) {
+        switch event {
+        case .editorSessionTemplateApply(let template):
+            editorSession.apply(template: template)
+        case .editorSessionTemplatePreview(let template):
+            WPAnalytics.track(.editorSessionTemplatePreview, withProperties: ["template": template])
```

Comment:
We should move this into editorSession and ensure all the other common properties are attached to this event as well.